java/test/src/main/java/test/Ice/background/EndpointI.java

```diff
@@ -4,6 +4,8 @@
 
 package test.Ice.background;
 
+import java.util.stream.Collectors;
+
 final class EndpointI extends com.zeroc.Ice.EndpointI {
```

Comment:
You could also then get rid of this import:
```suggestion
package test.Ice.background;

final class EndpointI extends com.zeroc.Ice.EndpointI {
```